internal/controller/api/v1alpha1/agentserviceconfig_types.go

```diff
@@ -26,10 +26,15 @@ import (
 type AgentServiceConfigSpec struct {
 	// FileSystemStorage defines the spec of the PersistentVolumeClaim to be
 	// created for the assisted-service's filesystem (logs, etc).
+	// With respect to the resource requests, the amount of filesystem storage
+	// consumer will depend largely on the number of clusters you intend to create.
+	// It's best to request as much as possible with the minimum being 20GiB in
+	// development/testing and 100Gib anywhere else.
 	//+operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Storage for service filesystem"
 	FileSystemStorage corev1.PersistentVolumeClaimSpec `json:"filesystemStorage"`
 	// DatabaseStorage defines the spec of the PersistentVolumeClaim to be
 	// created for the database's filesystem.
+	// With respect to the resource requests, 10Gi is sufficient.
```

Comment:
It would be nice to use the current size of the DB in `cloud.redhat.com` as a reference. I've asked how big it is already, waiting on an reply

Reply:
I probably also shouldn't be saying that 10Gi is sufficient....maybe that it's the minimum recommended.